games-action/prismlauncher/prismlauncher-9999.ebuild

```
@@ -118,6 +118,7 @@ src_configure(){
 		-DCMAKE_INSTALL_PREFIX="/usr"
 		# Resulting binary is named prismlauncher
 		-DLauncher_APP_BINARY_NAME="${PN}"
+		-DLauncher_BUILD_PLATFORM="Gentoo"
```

Would it be useful to do something like:
```suggestion
		-DLauncher_BUILD_PLATFORM="Gentoo ${PF}"
```
so it has the ebuild revision in it too? Or not needed?

---

I think it's not needed, since the variable is only about the build platform. If there are any issues with our packaging later down the line, it will be enough information for upstream to send the issue on our way, and then we can follow the usual procedure of bumping the revision once a fix is made. :slightly_smiling_face: